support/README.md

```diff
@@ -115,13 +115,14 @@ should:
 Over time, we want to be able to track the overall support load on the
 exterminator so we can adjust the process. We use two main metrics:
 
-- The exterminator handoff form asks for a count of how many unique users 
-  asked a question in the Slack channel each day.
+- The exterminator handoff form asks for a count of how many unique users
+  filed a Github issue or asked a question in the Slack channel each day.
+  (This number should not reflect Tilters opening Github issues.)
```

Comment:
What if a Tilter opens a GitHub issue on behalf of a user?

Reply:
Gooood question. I thiiiink we care if the EXTERMINATOR opens an issue on behalf of a user (though often that user will have some through the slack channel) because the point of these numbers is to measure support load for the exterminator. So I'd propose reporting: # unique users asking Qs in channel/opening GH issues/reporting issues to the exterminator via some other channel such that the exterminator opens an issue for them. does that sound about right?

Reply:
V happy for other suggestions here, I just wanted to make sure that "# new github issues" got included in the exterminator metrics somehow b/c it's a measure of how much support the exterminator has to give.

Reply:
Thanks for the explanation @maiamcc . I think the context  already makes your addition clear, especially for an exterminator. I just made a suggestion of the wording. Closing this thread. You can take a look there to see if you want to accept that.